nf_core/__main__.py

```diff
@@ -1622,6 +2014,56 @@ def bump_version(new_version, dir, nextflow):
         sys.exit(1)
 
 
+# nf-core create-logo
+
+
+@nf_core_cli.command("create-logo")
+@click.argument("logo-text", metavar="<logo text>")
+@click.option("-d", "--dir", type=click.Path(), default=".", metavar="<directory>")
+@click.option(
+    "--theme",
+    type=click.Choice(["light", "dark"]),
+    default="light",
+    help="Theme for the logo.",
+    show_default=True,
+)
+@click.option(
+    "--width",
+    type=int,
+    default=2300,
+    help="Width of the logo in pixels. Will be overwritten if `logo-text` is longer than the given width.",
+    show_default=True,
+)
+@click.option(
+    "--no-prompts",
+    is_flag=True,
+    help="Do not confirm changes, just update parameters and exit",
```

Comment:
What parameters?

Reply:
I guess this is a copy-pasta. Could maybe be dropped, and also removed from the Class as I don't think it's used?